### PR TITLE
DIV-5382: Separated Petitioner & Respondent journeys

### PR DESCRIPTION
**Pull request by johnoneill46**
Split petitioner and respondent into their own separate journeys so as the DA app expands it will be easier to manage. 
- Added logic to infer who is logging into DA; we then use this to decide which path to show
- Copied over simple tests

All current e2e/functional tests etc pass for the petitioner journey.

### CODE REVIEW

File: test/unit/steps/common/entry.test.js [hunk 1]
```diff
@@ -26,10 +28,12 @@ describe(modulePath, () => {
       caseOrchestrationHelper.handleErrorCodes.restore();
     });
 
+    /* Need to readd
```
Comment (Dizrupto) [inline test/unit/steps/common/entry.test.js]: Does anyone know whether this needs to be added back on this PR, or later on?

File: steps/petitioner/progress-bar/PetitionerProgressBar.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{% extends "progress-bar.njk" %}
+
+{% block main_content %}
+    <p class="govuk-body-s">
+        <strong> {{ content.referenceNumber }}:</strong>
```
Comment (yarpo) [inline steps/petitioner/progress-bar/PetitionerProgressBar.html]: space between `<strong>` and `{{` has any purpose?

File: steps/petitioner/progress-bar/PetitionerProgressBar.step.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  get stateTemplate() {
+    if (this.currentCaseState.toLowerCase() === caseStates.divorceGranted) {
+      return progressBarMap.fourCirclesFilledIn;
+    }
+    return progressBarMap.threeCirclesFilledIn;
+  }
+
+  get pageContentTemplate() {
+    let pageContent = '';
+    /* eslint-disable indent */
```
Comment (yarpo) [inline steps/petitioner/progress-bar/PetitionerProgressBar.step.js]: why?

File: steps/respondent/progress-bar/RespondentProgressBar.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{% extends "progress-bar.njk" %}
+
+{% block main_content %}
+    <p class="govuk-body-s">
+        <strong> {{ content.referenceNumber }}:</strong>
```
Comment (yarpo) [inline steps/respondent/progress-bar/RespondentProgressBar.html]: as above

File: test/unit/steps/common/entry.test.js (hunk 1 above)
Comment (yarpo) [inline test/unit/steps/common/entry.test.js]: why?

File: steps/respondent/progress-bar/RespondentProgressBar.html (hunk 4 above)
Comment (charlesluokb) [reply steps/respondent/progress-bar/RespondentProgressBar.html]: extra space removed

File: test/unit/steps/common/entry.test.js (hunk 1 above)
Comment (charlesluokb) [reply test/unit/steps/common/entry.test.js]: removed the block being commented out

File: steps/petitioner/progress-bar/PetitionerProgressBar.step.js (hunk 3 above)
Comment (charlesluokb) [reply steps/petitioner/progress-bar/PetitionerProgressBar.step.js]: enabled the annotation. However @ravinderpal  made me aware of that the current acceptable indentation is not right and he will raise a tech debt to address it.

File: steps/petitioner/progress-bar/PetitionerProgressBar.html (hunk 2 above)
Comment (charlesluokb) [reply steps/petitioner/progress-bar/PetitionerProgressBar.html]: removed extra space

**Comment by nsingamsetty** [comment]
This is tested and refer the evidences DIV-5382. Merging now.

**Pull request merged**